frappe/public/js/frappe/views/calendar/calendar.js

```diff
@@ -227,6 +227,7 @@ frappe.views.Calendar = Class.extend({
 			defaultView: defaults.defaultView,
 			weekends: defaults.weekends,
 			nowIndicator: true,
+			now: frappe.datetime.convert_to_system_tz(),
```

Comment:
Not sure if this is the correct solution since it gives priority to system timezone over the machine timezone.